client/stack-editor/lib/editor/markdowneditorview.coffee

```diff
@@ -9,3 +11,36 @@ module.exports = class MarkdownEditorView extends BaseStackEditorView
     options.targetContentType ?= 'markdown'
 
     super options, data
+
+
+  viewAppended: ->
+
+    super
+
+    @addSubView new KDButtonView
```

Comment:
MarkdownEditorView has already `Preview` button. It is created [here](https://github.com/koding/koding/blob/0efae6358c24b325571c543e6c637c6cfbd733c7/client/stack-editor/lib/editor/index.coffee#L264)

And event to show readme preview is listened [here](https://github.com/koding/koding/blob/0efae6358c24b325571c543e6c637c6cfbd733c7/client/stack-editor/lib/editor/index.coffee#L405)
And modal is shown [here](https://github.com/koding/koding/blob/0efae6358c24b325571c543e6c637c6cfbd733c7/client/stack-editor/lib/editor/index.coffee#L859)

So this part is unneccesary, no need to add the button and its handler function